src/material/button/_button-theme.scss

```diff
@@ -270,6 +294,20 @@
         @include _outlined-button-variant($theme, warn);
       }
     }
+
+    .mat-tonal-button {
```

Comment:
Should we support this in M2 themes? just feels like extra CSS bloat. I think we could say its an M3 only feature - we already have a few appearance options like that if I recall

Reply:
The M2 spec doesn't have tonal buttons so we don't really support it. It just looks like a filled button.